github/config/mt-github/500-adapter-ksvc.yaml

```diff
@@ -21,6 +21,8 @@ metadata:
     contrib.eventing.knative.dev/release: devel
     sources.knative.dev/role:    adapter
     eventing.knative.dev/source: github-source-controller
+    # Add this line when the vanity URL experimental feature is enabled.
+    # serving.knative.dev/visibility: cluster-local
```

Comment:
If we know this line is needed for the feature -- is there a way we could perhaps add it programmatically?